.github/CONTRIBUTING.md

```diff
@@ -53,9 +53,9 @@ make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool_basic'
 
 The `TESTARGS` variable is regexp-like, so multiple tests can be run in parallel by specifying a common substring of those tests (for example, `TestAccContainerNodePool` to run all node pool tests). There are 1500+ tests, and running all of them takes over 4 hours and requires a lot of GCP quota.
 
-### Ensuring no plan-time difference to upstream provider
+### Ensuring no plan-time difference to published provider
 
-To ensure the resource you are modifying doesn't result in diff to existing deployments you can run set the [environment variable](https://github.com/GoogleCloudPlatform/magic-modules/blob/a30da2040ca7b8bd37186d8521a911e7469da632/mmv1/third_party/terraform/utils/provider_test.go.erb#L284-L286) `UPSTREAM_DIFF` before running a test. This will append plan only steps using the upstream provider (`google` or `google-beta`) after all configuration deployments to ensure uniformity. This is recommended when you are modifying existing fields and do not want to break existing user deployments.
```

Comment:
Not a requirement for this change, just a consideration: This documentation answers the "what" this step does, and the "how", but not the "why"- we've got a lot of context as the provider maintainers- but consider someone who's onboarding or contributing as a community member. _Why_ might they want to follow these steps? For example, under what circumstances is performing this type of testing useful?

Reply:
added why to the beginning and removed rationalization from the end